app/_data/docs_nav_konnect.yml

```diff
@@ -305,6 +315,14 @@
           target_blank: true
         - text: Audit Log Webhooks
           url: /api/audit-logs/webhook/
+    - text: Mesh Manager API
+      items:
+        - text: API Documentation
+          url: https://deploy-preview-252--kong-platform-api.netlify.app/konnect/mesh-control-planes/v0/
```

Comment:
Is this right? Shouldn't it use a normal url like /api/konnect/mesh-control-planes?